orttraining/orttraining/test/python/orttraining_test_ortmodule_autograd.py

```diff
@@ -53,7 +53,10 @@ def bias_gelu_backward(g, bias, y):
     class GeLUFunction1(torch.autograd.Function):
         @staticmethod
         def forward(ctx, input, bias):
-            ctx.save_for_backward(input, bias)
+            # Clone the tensor inputs of forward function since those tensors may be allocated by ORT
+            # and will be released after forward function returns (since ORT is not aware of the reference
+            # in ORT allocation plan).
+            ctx.save_for_backward(input.detach().clone(), bias.detach().clone())
```

Comment:
This could introduce a perf regression. Is it possible to convert them into torch tensor using dlpack and then save them. That way, torch would be the owner of the tensors and there would not be any copy.